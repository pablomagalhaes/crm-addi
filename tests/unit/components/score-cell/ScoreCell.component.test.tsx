import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import ScoreCell from '@/components/score-cell/ScoreCell.component';

describe('<ScoreCell />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<ScoreCell score={60} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should have the score number', () => {
    const wrapper = shallow(<ScoreCell score={60} />);
    expect(wrapper.text()).toContain('60');
  });
});
