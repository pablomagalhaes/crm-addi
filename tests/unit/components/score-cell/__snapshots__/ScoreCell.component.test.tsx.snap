// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScoreCell /> should match snapchat 1`] = `
<styled.div>
  <styled.span
    styleType="success"
  >
    60
  </styled.span>
</styled.div>
`;
